### PR TITLE
New home work

### PR DESCRIPTION
**Pull request by Daniluko**
Look at HomeWork

### CODE REVIEW

File: Lesson10/.idea/workspace.xml [hunk 1]
```diff
@@ -2,35 +2,16 @@
 <project version="4">
   <component name="ChangeListManager">
     <list default="true" id="2db53d9c-a718-4ea5-a466-83f1f3abfd9d" name="Default Changelist" comment="">
-      <change afterPath="$PROJECT_DIR$/../HomeWork/.idea/compiler.xml" afterDir="false" />
```
Comment (SergeyPasko) [inline Lesson10/.idea/workspace.xml]: Many times I say about this

File: Lesson10/.idea/dictionaries/rostyslav.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<component name="ProjectDictionaryState">
```
Comment (SergeyPasko) [inline Lesson10/.idea/dictionaries/rostyslav.xml]: ?

File: Lesson10/src/main/java/Lesson11/StringChangerService.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package Lesson11;
+
+/**
+ * Created by Rostyslav on 28.01.19
+ */
+public class StringChangerService {
```
Comment (SergeyPasko) [inline Lesson10/src/main/java/Lesson11/StringChangerService.java]: Why I see that in new homeWork?

File: Lesson10/src/main/java/lessonnew13/MateGroupServiceImpl.java [hunk 4]
```diff
@@ -22,12 +23,20 @@
...
         return Response.status(javax.ws.rs.core.Response.Status.OK).entity(mateGroup).type(MediaType.APPLICATION_JSON).build();
     }
-
-
     @PUT
-    @Path("/{groupId}")
+    @Override
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
+    @Path("/{groupId}")
```
Comment (SergeyPasko) [inline Lesson10/src/main/java/lessonnew13/MateGroupServiceImpl.java]: bad path names. How you undestand in rest what you doinig?

Comment (Daniluko) [reply Lesson10/src/main/java/lessonnew13/MateGroupServiceImpl.java]: Yes, i update this

Comment (Daniluko) [reply Lesson10/src/main/java/lessonnew13/MateGroupServiceImpl.java]: Look all commits

File: Lesson10/src/main/java/lessonnew13/MateGroupServiceClient.java [hunk 5]
```diff
@@ -52,7 +55,7 @@ private static void showInfo(HttpURLConnection conn) throws IOException {
 
     private static HttpURLConnection addStudent(String pathParam)
             throws MalformedURLException, IOException, ProtocolException {
-        URL url = new URL("http://localhost:9998/rs/mate/" + pathParam);
+        URL url = new URL("http://localhost:9998/rs/mate/" + pathParam+"/teacher");
```
Comment (SergeyPasko) [inline Lesson10/src/main/java/lessonnew13/MateGroupServiceClient.java]: Crazy view

File: Lesson10/src/main/java/lessonnew13/MateGroupServiceImpl.java [hunk 6]
```diff
@@ -22,12 +23,20 @@
...
     }
-
-
     @PUT
-    @Path("/{groupId}")
+    @Override
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
+    @Path("/{groupId}/teacher")
+    public Response updateTeacher(@PathParam("groupId") int groupId, Teacher teacher) {
```
Comment (SergeyPasko) [inline Lesson10/src/main/java/lessonnew13/MateGroupServiceImpl.java]: You nothing doing with input param teacher

**Pull request merged**